MicrosoftTeams.schema.json

```diff
@@ -805,6 +805,24 @@
       },
       "description": "When a group install scope is selected, this will define the default capability when the user installs the app",
       "additionalProperties": false
+    },
+    "configurableProperties": {
+      "type": "array",
+      "description": "A list of tenant configurable properties for an app",
+      "maxItems": 9,
+      "items": {
+        "enum": [
+          "name",
+          "shortDescription",
+          "longDescription",
+          "smallImageUrl",
```

Comment:
See the section below how they group the image urls to the grouping of "icons"  we should follow that same pattern.

Reply:
> See the section below how they group the image urls to the grouping of "icons" we should follow that same pattern.

I just want the list of these enum to be in the array to indicate they are configurable, grouping will make it complicated